host-bin/startxiwi

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh -e
+# Copyright (c) 2015 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+APPLICATION="${0##*/}"
+OPTS_ENTER=''
+OPTS_XIWI=''
+
+USAGE="$APPLICATION [options] chroot_app [parameters]
+
+Wraps enter-chroot to launch a window or tab in Chromium OS for any graphical application.
+Applications launched in this way show in independent windows or tabs.
+
+By default, it will use the primary user on the first xiwi-enabled chroot found.
+
+Enter '-h' for help with 'enter-chroot' and 'xiwi' options.
+"
+
+USAGE_HELP="Usage: enter-chroot [options]
+
+`enter-chroot -h 2>&1 | grep -e ' -[bckntu]'`
+
+    Options: -l, -x, and -X are deprecated for the xiwi application.
+
+
+Usage: xiwi [options] chroot_app [parameters]
+
+`enter-chroot -t xiwi xiwi -h 2>&1 | grep -e ' -[FT]' | sed 's/^/    /g'`
```

Comment:
That, unfortunately, requires root, and may give different results depending on which chroot gets used in which instance.  I think for now, just copy-paste the text from xiwi, and when the rebase/parts stuff gets merged we can dedupe the text.